src/main/resources/application.properties

```diff
@@ -2,12 +2,12 @@
 puredata.patch.path=/Users/giuseppedesiato/Documents/Pd/sine_tone.pd
 
 # For MySQL
-spring.datasource.url=jdbc:mysql://localhost:3306/puresynth
-spring.datasource.username=${DB_USERNAME}
-spring.datasource.password=${DB_PASSWORD}
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
-spring.jpa.hibernate.ddl-auto=update
-logging.level.org.springframework.data=DEBUG
+#spring.datasource.url=jdbc:mysql://localhost:3306/puresynth
```

Comment:
Why did you comment this out? It will be needed by the app. This config is for the app to know, how to connect to the DB. The `docker-compose` just creates the database.